gradle/libs.versions.toml

```diff
@@ -0,0 +1,142 @@
+[versions]
```

Comment:
this file has been somehow generated?

Reply:
I write it myself, I wrote a [blogpost](https://funkymuse.dev/posts/toml-gradle/) explaining how it works.

Reply:
Although it will become the norm with Android Studio Hedgehog and above